ui/src/keyboardLayouts/da_DK.ts

```diff
@@ -163,6 +163,12 @@ export const chars = {
   "_": { key: "Slash", shift: true },
   "<": { key: "IntlBackslash" },
   ">": { key: "IntlBackslash", shift: true },
+  "~": { key: "BracketRight", deadKey: true, altRight: true }, 
+  "^": { key: "BracketRight", deadKey: true, shift: true }, 
+  "¨": { key: "BracketRight", deadKey: true, }, 
+  "|": { key: "Equal", deadKey: true, altRight: true},
```

Comment:
Hmmm, sorry if I steered you wrong, this looks to be the correct location (and that it's a dead key) per the layout I posted, but this conflicts with line 118 which says that the `|` character lives on the backquote key.

According the layout I posited, that key [has `§` (shifted) and `½` (unshifted)](https://github.com/jetkvm/kvm/pull/839/files/51345670c6c1ea401328b4f19c3136751b2b0336..654e965bb015d9da5834a11ee8d4a379aaca7b02#diff-0673711686cc281ed7d00489c7df3527bc6193e37f7526823baa762e38ee95d0R118-R119), so I think you _also_ need to change line 118 to read `"½": { key: "Backquote" },`  but I'm not Danish ;)